tests/hooks/test_firebolt.py

```diff
@@ -51,6 +51,7 @@ def test_get_conn(self, mock_connect):
             api_endpoint="api_endpoint",
             database="firebolt",
             engine_name="test",
+            account_name="firebolt"
```

Comment:
lets add a test for `engine_name=None` as well.

Reply:
will add